compiler/natives/src/reflect/reflect.go

```diff
@@ -491,7 +491,8 @@ func makechan(typ *rtype, size uint64) (ch unsafe.Pointer) {
 	return unsafe.Pointer(js.Global.Get("$Chan").New(jsType(ctyp.elem), size).Unsafe())
 }
 
-func makemap(t *rtype) (m unsafe.Pointer) {
+func makemap(t *rtype, cap int) (m unsafe.Pointer) {
+	// TODO: Use cap if needed/possible.
```

Comment:
I think there is no way to properly use `cap` here, so the comment can be removed.